constants.py

```diff
@@ -728,7 +735,7 @@
 
 kAngleMotorCANId = 50
 kAngleMotorName = "ShooterAngleMotor"
-kAngleMotorPGain = 0.1
+kAngleMotorPGain = 0.9
```

Comment:
arbitrary?

Reply:
yeah, needs to be tuned on actual machine, was done to make snappier sim